ns/namespace.go

```diff
@@ -143,11 +143,19 @@ func GetValueByNamespace(object interface{}, ns []string) interface{} {
 					return GetValueByNamespace(val.MapIndex(kval).Interface(), ns[2:])
 				}
 			case reflect.Ptr:
-				val = reflect.ValueOf(val).Elem().Interface()
+				v := reflect.Indirect(reflect.ValueOf(val))
+				if !v.IsValid() || !v.CanInterface() {
+					return v
```

Comment:
Hm, it might not be enough to return v (type reflect.Value) here. Seeing the following intermittent failure with Snap tasks:

```
gob: type not registered for interface: reflect.Value
```

Perhaps `return val` (make no changes) might make more sense here, but is that safe?